### ISSUE TITLE
learned ntp synchronization on Cisco Catalyst 9407R switches is not accurate.

### ISSUE THREAD

**Issue by nathanielfernandez** [CiscoTestAutomation/genielibs#98]
I noticed that my script learned ntp and it says 'unsynchronized', but when i checked the device via ssh it is actually synchronized. I've seen this occur on Cisco Catalyst 9407R switches (ios-xe) in my testbed.
![f0574923-1262-4ae1-855b-682a65b69851](https://user-images.githubusercontent.com/93752762/159381656-9189a351-a2e6-409e-b69d-b321a7eb1c4c.PNG)
![50a21a84-605c-4e33-8e0b-b869bc5c1151](https://user-images.githubusercontent.com/93752762/159381662-eff4d895-3abd-4428-89b6-050ff59ebe9d.PNG)

**Comment by LukasMcClelland** [comment]
Thank you bringing this to our attention. Since genieparser is a community driven repo, we encourage users to make fixes for smaller errors themselves. Would you be willing to fix this and then open up a PR to commit the changes? If not, that's alright, we can handle it if you choose not to

**Comment by nathanielfernandez** [comment]
Hey Lukas. This is noted. I would like to solve this one to become more familiar with the structure. Just give me enough time to learn how to fix and test. :)

**Comment by LukasMcClelland** [comment]
Of course! Happy to hear that you'll take on the challenge yourself. For help in parser related fixes we have this page
https://pubhub.devnetcloud.com/media/pyats-development-guide/docs/writeparser/writeparser.html

I'll close the issue for now. But please feel free to reopen it if necessary
